force-app/main/default/classes/SOQL.cls

```diff
@@ -163,9 +254,17 @@ public with sharing class SOQL {
      */
     public static Decimal getTotalRevenueByStage(String stage) {
         // Use AggregateQuery to get the total revenue
+        aggregateResult[] results = [SELECT SUM(Amount) totalRevenue FROM Opportunity WHERE StageName = :stage OR amount > 10000];
+
+        Decimal totalRevenue = 0.0;
+
+        if(results.size() > 0) {
```

Comment:
this is a good check here, though, because we may have 0 Opps that fit our WHERE clause